guides/privacy-faq.md

```diff
@@ -38,7 +38,7 @@ Here's an example of how you could mention TelemetryDeck in your privacy policy:
 
 ## Do I need to fill out Apple's app privacy details?
 
-By default, the privacy manifest metadata included in the TelemetryDeck Swift SDK should already fill out Apple App Privacy Details information for you. See our [App Privacy Guide](/docs/articles/apple-app-privacy/) for more information.
+Yes, every app needs to fill out this form, even those that do not track any data. But our Swift SDK includes a privacy manifest which should list all the right things in your [generated privacy report](https://developer.apple.com/documentation/bundleresources/describing-data-use-in-privacy-manifests#Create-your-apps-privacy-report). Alternatively, see our [App Privacy Guide](/docs/articles/apple-app-privacy/) for detailed instructions.
```

Comment:
This seems platform-specific, perhaps it can be reworded so it refers to a general marketplace where the app will be available.

We can still provide examples with the App Store as it's shown above, but also Google Play's [Data Safety](https://support.google.com/googleplay/android-developer/answer/10787469?hl=en-GB) report, Meta's [Data Use Checkup](https://developers.meta.com/horizon/resources/publish-data-use) and others (if needed).

Reply:
@kkostov The section here specifically answers the question for "Apple's app privacy". If there are similarly required forms/fields to fill out for other stores, feel free to create a new PR (after I merge this) that adds additional questions about these other stores. I think, since this is an FAQ and the stores work differently, it should have each "use case" answered separately.